src/shared/components/palette/case-link/components/linked-cases-table/linked-cases-to-table.component.ts

```diff
@@ -82,8 +82,8 @@ export class LinkedCasesToTableComponent implements OnInit, AfterViewInit {
     let secondLevelresultArray = [];
     const data = this.caseField && this.caseField.value || [];
     data.forEach((item: any) => {
-      const progressedStateReason = item.reasons.find(reason => reason.reasonCode === 'Progressed')
-      const consolidatedStateReason = item.reasons.find(reason => reason.reasonCode === 'Case consolidated')
+      const progressedStateReason = item.reasons.find(reason => reason.reasonCode === 'CLRCO16') // PROGRESSED AS A LEAD CASE
+      const consolidatedStateReason = item.reasons.find(reason => reason.reasonCode === 'CLRCO15') // CASE CONSOLIDATED
```

Comment:
is there any business logic to hard code and specially handle these two reasonCodes

Reply:
yes @rayliangatos based on the discussion with Adeel only those two reason codes needs to be considered.